sstables/sstables.hh

```diff
@@ -616,6 +624,8 @@ private:
     void validate_partitioner();
 
     void set_first_and_last_keys();
+    // Retrieves clustering range for first and last keys using the promoted index.
```

Comment:
How it is done is out of scope for the api-level comment.
why does the caller care about how the function is implemented internally?

Maybe `set_ ...` is not not the right name for it. If it could be so, I'd expect it to get the first and last ranges as parameters.  Bit since we already have `set_first_and_last_keys`, let's keep the convention.

Reply:
You're right, but the damage is limited since it's a private function. The naming did confuse me earlier.

I suggest to rename them later to load_first_and_last_keys() since we're reading from persistent storage.